lib/tasks/router_data.rake

```diff
@@ -11,9 +11,9 @@ namespace :router_data do
 
     puts "Writing file with #{artefacts.count} redirects"
 
-    csv = "Source,Destination,Type\n"
+    csv = "Source,Destination,Type,Segments\n"
     csv << artefacts
-      .map { |arr| "/#{arr[0]},#{arr[1].gsub(/#.*/, '')},prefix" }
+      .map { |arr| "/#{arr[0]},#{arr[1]},prefix" }
```

Comment:
I think you need to swap the positions of `arr[1]` and `prefix` for this to work